pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -30,8 +30,7 @@
   isPy3k, pythonOlder }:
 
 # assert that everything needed for cuda is present and that the correct cuda versions are used
-assert !cudaSupport || (let majorIs = lib.versions.major cudatoolkit.version;
-                        in majorIs == "9" || majorIs == "10" || majorIs == "11");
+assert !cudaSupport || lib.versionAtLeast cudatoolkit.version "10.2.0";
```

Comment:
It is an assertion of pytorch derivation, not pytorch-bin. I don't think it's necessary to put it back.

Reply:
I meant to drop this, thanks